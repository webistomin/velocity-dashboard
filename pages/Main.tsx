import { VueComponent } from 'types/vue-components';
import { Component } from 'nuxt-property-decorator';
import { VNode } from 'vue';

@Component({
  name: 'MainPage',
})
export default class MainPage extends VueComponent {
  render(): VNode {
    return (
      <main class='page-content'>
        <h1>Main</h1>
      </main>
    );
  }
}
