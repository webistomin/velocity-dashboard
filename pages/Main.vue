<template lang="pug">
  h1 Main
</template>

<script lang="ts">
  import Vue from 'vue';

  export default Vue.extend({});
</script>
