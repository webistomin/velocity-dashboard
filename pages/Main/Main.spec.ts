import { shallowMount } from '@vue/test-utils';
import Main from './Main';

describe('Main page', () => {
  const factory = (propsData = {}, options = {}) => {
    return shallowMount(Main, {
      propsData: {
        ...propsData,
      },
      ...options,
    });
  };

  it('Matches snapshot', () => {
    const wrapper = factory().html();
    expect(wrapper).toMatchSnapshot();
  });
});
