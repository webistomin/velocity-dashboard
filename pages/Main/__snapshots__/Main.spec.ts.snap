// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page Matches snapshot 1`] = `
"<main class=\\"page-content page-content_offset\\">
  <overview-stub></overview-stub>
</main>"
`;
